ansible/install.yml

```diff
@@ -16,6 +16,17 @@
        - python-pip
        - iptables
        - software-properties-common
+       - qemu-kvm
+       - libvirt-bin
+       - libvirt-dev
+       - python-paramiko
+       - python-libvirt
+
+   - name: disable apparmor libvirt profile (to allow attaching of interfaces to running vms)
```

Comment:
That will not work in the Travis CI environment. What about having this optional so travis could test at least all "normal" features of the libvirt_support branch without the libvirt parts? @xschlef